### ISSUE TITLE
Broken npm package ?

### ISSUE THREAD

**Issue by ArnaudPel** [burgaard/string-algorithms#13]
## Import
As of version `1.0.21`, the documented import syntax leads to an error:
```js
import { longestCommonSubstring } from 'string-algorithms';
```

The following syntax does, however, work:
```js
import { longestCommonSubstring } from 'string-algorithms/src';
```

## Package structure
Examination of the package structure as found in `node_modules/` reveals the following differences between `1.0.20` (left) and `1.0.29` (right):

![Changes brought by 1 0 29 to string-algorithms package structure](https://user-images.githubusercontent.com/6020369/80974901-afa34800-8e21-11ea-8a2b-d8584faf6972.png)

The absence of the `lib/` directory in `1.0.29` seems to contradict [the following configuration in package.json](https://github.com/burgaard/string-algorithms/blob/3b7bcbb931c3d87e15bce28148971ecc571b58a7/package.json#L31)

## Tar exploration
As per [this answer](https://stackoverflow.com/a/33549885/), a direct download of [the tar from npm](https://registry.npmjs.org/string-algorithms/-/string-algorithms-1.0.29.tgz) confirms the absence of the `lib/` folder, which seems to rule out any issue with `npm install`

**Comment by ArnaudPel** [comment]
A bit more on this, as a comment.

I have looked into the git history since `1.0.20`, and couldn't find anything looking remotely like a suspect in this issue. Since `js` packaging standards still resist my full understanding, I'm afraid I won't be able to help much on this in the near future.

Lastly, a note on something that I have been dreaming of for our own CI pipelines: the practice of [running tests on the actual package that is going to end up on npm](https://medium.com/@vcarl/problems-with-npm-link-and-an-alternative-4dbdd3e66811). In case that's of any interest to you.

Cheers

**Comment by raine** [comment]
The module is broken.

package.json says main entrypoint is `lib/index.js`, but the published tarball does not have `lib` directory.

**Comment by burgaard** [comment]
Yikes. Thanks @ArnaudPel and @raine! Sorry for the long delay -- I use the same GitHub account for work, but for some reason I'm completely missing notifications about activity on my personal repos.

**Comment by burgaard** [comment]
@ArnaudPel, @raine, the issue should be resolved with version 1.0.31. My NPM integration in Travis CI had fallen behind, so commits weren't getting published. I think I probably ran a manual npm publish for 1.0.21 after realizing the package didn't update in npm.
